src/main/java/edu/ucsb/mapache/advice/AuthControllerAdvice.java

```diff
@@ -8,6 +8,7 @@
 import edu.ucsb.mapache.entities.AppUser;
```

Comment:
Not sure why it isn't working... but it didn't work for me when I tested it.
LMK if you want to meet up to try to debug

Reply:
But since this team already has reached the points goal, we might wait until after we've helped the teams that are behind.